Editor/Scripts/CodeGenerators/CodeGeneratorUtils.cs

```diff
@@ -0,0 +1,12 @@
+using System.Text.RegularExpressions;
+
+namespace RegressionGames
+{
+    public class CodeGeneratorUtils
+    {
+        public static string SanitizeActionName(string name)
+        {
+            return Regex.Replace(name.Replace(" ", "_"), "[^0-9a-zA-Z_]", "_");
```

Comment:
Good catch! Not sure if we need this now, but we may still get invalid names, like an action or state that is entirely numbers or starts with a number.

Reply:
Ah ok I see, thanks!